tests/test_utilities.py

```diff
@@ -101,12 +101,26 @@ def test_boxes_to_shapefile_projected_from_predict_tile(m):
     gdf = utilities.boxes_to_shapefile(df.iloc[:1,], root_dir=os.path.dirname(img), projected=True)
     assert gdf.shape[0] == 1
     
+# Test unprojected data, including warning if flip_y_axis is set to True, but projected is False
 @pytest.mark.parametrize("flip_y_axis", [True, False])
-def test_boxes_to_shapefile_unprojected(m, flip_y_axis):
+@pytest.mark.parametrize("projected", [True, False])
+def test_boxes_to_shapefile_unprojected(m, flip_y_axis, projected):
     img = get_data("OSBS_029.png")
     r = rio.open(img)
     df = m.predict_image(path=img)
-    gdf = utilities.boxes_to_shapefile(df, root_dir=os.path.dirname(img), projected=False, flip_y_axis=flip_y_axis)
+    if projected and flip_y_axis:
```

Comment:
I think I am missing something. Shouldn't this be if `not` ( either projected or Flip_y_axis)

Reply:
No I think thats right.
```
    if projected and flip_y_axis:
        with pytest.warns(UserWarning):
            gdf = utilities.boxes_to_shapefile(
                df,
                root_dir=os.path.dirname(img),
                projected=projected,
                flip_y_axis=flip_y_axis)
```

If the user sets projected to be True and flip_y_axis is True, it should raise a warning. Projected says the boxes are in geographic projection, there is no situation in which you would want to reflect over the y axis and get negative coordinates. The flip_y_axis is only useful in unprojected data and for programs like QGIS that have the origin flipped compared to numpy.